network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -626,10 +626,10 @@ def get_context(self, request, *args, **kwargs):
         context['categories'] = BuyersGuideProductCategory.objects.filter(hidden=False)
         context['mediaUrl'] = settings.CLOUDINARY_URL if settings.USE_CLOUDINARY else settings.MEDIA_URL
         context['use_commento'] = settings.USE_COMMENTO
-        context['pageTitle'] = f'''{pgettext(
-          'This can be localized. This is a reference to the “*batteries not included” mention on toys.',
-          '*privacy not included'
-        )} - {self.title}'''
+        context['pageTitle'] = f'{self.title} | ' + pgettext(
+                "This can be localized. This is a reference to the “*batteries not included” mention on toys.",
+                "Privacy & Security Guide"
+            ) + ' | Mozilla Foundation'
```

Comment:
Had to go with this because

<img width="960" alt="image" src="https://user-images.githubusercontent.com/2896608/107292122-28bd7480-6a1e-11eb-8d6a-8650688d37c0.png">

And if I use `f'text'` and put everything in one line, flake8 will complain that the line being too long. 🤔

Reply:
python lets you break up strings over multiple lines by putting it in parens and then splitting it:
```
thing = "if this would be a long line then the following format will split it:"

longersplit = (
  "for example this is a long line"
  " and can be broken up with just"
  " a few more quote symbols"
)
```

Reply:
Ahhhhh alright! Thanks!